spec/openapi.yml

```diff
@@ -1424,6 +1427,9 @@ components:
           type: array
           items:
             $ref: '#/components/schemas/Job'
+        totalCount:
+          type: number
+          description: The total number of jobs for the given namespace
 
     CreatedRun:
```

Comment:
Should we also document how to use the `limit` and `offset` for offset-based pagination?